doc/man3/RSA_public_encrypt.pod

```diff
@@ -46,17 +48,21 @@ Encrypting user data directly with RSA is insecure.
 
 =back
 
-B<flen> must be less than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
-based padding modes, less than RSA_size(B<rsa>) - 41 for
+B<flen> must not be more than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
+based padding modes, not more than RSA_size(B<rsa>) - 42 for
 RSA_PKCS1_OAEP_PADDING and exactly RSA_size(B<rsa>) for RSA_NO_PADDING.
 The random number generator must be seeded prior to calling
 RSA_public_encrypt().
+B<to> and B<from> may overlap.
```

Comment:
be the same, not overlap?

Reply:
they may overlap.

Reply:
I guess overlap works with the current implementation, but I'm not sure we should document that.

Reply:
I use that property in my OPC/UA stack.